docs/getting-started.md

```diff
@@ -48,7 +48,7 @@ Now that your node is fully synced, you can use it to list all organizations tha
 are registered in the Radicle network.
 
 ```bash
-radicle-registry-cli orgs list --node-host localhost
+radicle-registry-cli org list --node-host localhost
```

Comment:
Is the `--node-host` option necessary? The other invocations don’t use it.

Reply:
not required, good point. Will update